### PR TITLE
am: add doc regarding Passwordless remember device option

### PR DESCRIPTION
**Pull request by tcompiegne**
closes gravitee-io/issues#8506

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/issues-8506-passwordless-ux-flow/index.html)

### CODE REVIEW

File: pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc [hunk 1]
```diff
@@ -58,6 +58,17 @@ NOTE: After the users complete the registration process, their authenticators ar
 
 image::{% link images/am/current/graviteeio-am-userguide-passwordless-authenticators.png %}[User authenticators]
 
+==== Remember device
+
+[label label-version]#New in version 3.18.8#
+
+To improve user experience, AM can determine if a passwordless device is already enrolled or not for a user and decide to prompt directly
+the passwordless login page the next time a user wants to sign-in.
```
Comment (ivan-gravitee) [inline pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc:66]:
```suggestion
the passwordless login page the next time a user wants to sign in.
```

File: pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc [hunk 2]
```diff
@@ -58,6 +58,17 @@ NOTE: After the users complete the registration process, their authenticators ar
 
 image::{% link images/am/current/graviteeio-am-userguide-passwordless-authenticators.png %}[User authenticators]
 
+==== Remember device
+
+[label label-version]#New in version 3.18.8#
+
+To improve user experience, AM can determine if a passwordless device is already enrolled or not for a user and decide to prompt directly
```
Comment (ivan-gravitee) [inline pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc:65]:
```suggestion
To improve user experience, AM can determine if a passwordless device is already enrolled (or not) for a user, and decide to prompt directly
```

File: pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc [hunk 3]
```diff
@@ -58,6 +58,17 @@ NOTE: After the users complete the registration process, their authenticators ar
...
+the passwordless login page the next time a user wants to sign-in.
+
+image::{% link images/am/current/graviteeio-am-userguide-passwordless-remember-device.png %}[Passwordless Remember Device]
+
+NOTE: This option has been based where users always use the same browser and have one mobile and one laptop/desktop computer.
```
Comment (ivan-gravitee) [inline pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc:70]: This is a bit unclear

**Review by ivan-gravitee — CHANGES_REQUESTED** [review]

File: pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc (hunk 3 above)
Comment (tcompiegne) [reply pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc]: Remove as it was unclear

File: pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc (hunk 2 above)
Comment (tcompiegne) [reply pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc]: Force pushed

File: pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc (hunk 1 above)
Comment (tcompiegne) [reply pages/am/3.x/user-guide/passwordless/user-guide-passwordless.adoc]: Force pushed

**Review by ivan-gravitee — APPROVED** [review]
LGTM

**Review by leleueri — APPROVED** [review]

**Pull request merged**